umad/src/main/java/com/pkukielka/AppConfig.java

```diff
@@ -8,6 +8,7 @@
     public int intervalMs = 100;
     public boolean shouldThrowExceptions = true;
     public boolean shouldPrintStackTrace = false;
+    public boolean shouldStoreSafeMethod = false;
```

Comment:
Why we need that?

Reply:
Tests

Reply:
That is what I thought, and I do not like it :P 
Let me look at the code again...

Reply:
I think it's unneeded. You can just check output to see if method was reported or not. And it's better end-to-end test also.